records_mover/db/postgres/copy_options/__init__.py

```diff
@@ -34,20 +37,59 @@ def needs_csv_format(hints: RecordsHints) -> bool:
     return False
 
 
-def postgres_copy_options(unhandled_hints: Set[str],
-                          load_plan: RecordsLoadPlan) -> Tuple[Optional[DateInputStyle],
-                                                               PostgresCopyOptions]:
+# unloading
+def postgres_copy_to_options(unhandled_hints: Set[str],
+                             delimited_records_format: DelimitedRecordsFormat,
+                             fail_if_cant_handle_hint: bool) ->\
+                                         Tuple[DateOutputStyle,
+                                               Optional[DateOrderStyle],
+                                               PostgresCopyOptions]:
+    hints = delimited_records_format.hints
+
+    if needs_csv_format(hints):
+        copy_options = postgres_copy_options_csv(unhandled_hints,
+                                                 hints,
+                                                 fail_if_cant_handle_hint,
+                                                 CopyOptionsMode.UNLOADING)
+    else:
+        copy_options = postgres_copy_options_text(unhandled_hints,
+                                                  hints,
+                                                  fail_if_cant_handle_hint,
+                                                  CopyOptionsMode.UNLOADING)
```

Comment:
Postgres uses the same fundamental COPY statement for both loading and unloading, but there is some nuance for some of the bits, so I'm passing in a flag here.